build/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+# This file controls the arguments shared by the build steps.
```

Comment:
Please move the binary and image building scripts to kubecf-tools.

Reply:
Will do. Not _quite_ sure yet how they'll work, though, since those are pretty repo-specific.

As a note: the binary (and image building) scripts are unused by CI. But they still make sense to have for local development (mostly), so I'll move them over.